client/src/java/com/zimbra/client/ZBaseItem.java

```diff
@@ -70,7 +78,24 @@ public String getUuid() {
 
     @Override
     public int getModifiedSequence() {
```

Comment:
What is modseq value of a message draft or a contact when it is first created and what it the modseq value of a non-draft message when it is first received? I wonder when modseq is actually going to be "0" and can be expected to increase.